tests/app/notifications/test_notifications_ses_callback.py

```diff
@@ -30,35 +30,24 @@ def test_process_ses_results_in_complaint(
     sample_notification,
 ):
     ref = str(uuid4())
-    notification = sample_notification(gen_type=EMAIL_TYPE, reference=ref)
+    notification: Notification = sample_notification(gen_type=EMAIL_TYPE, reference=ref)
```

Comment:
The annotation isn't helpful here.  The type is obvious.
```suggestion
    notification = sample_notification(gen_type=EMAIL_TYPE, reference=ref)
```